discovery-provider/src/queries/get_users_cnode.py

```diff
@@ -67,7 +72,10 @@ def get_users_cnode(cnode_endpoint_string, replica_type=ReplicaType.PRIMARY):
                     't.secondary1 = :cnode_endpoint_string OR '
                     't.secondary2 = :cnode_endpoint_string) AND'
             }
-            t.secondary1 is not NULL;
+            t.user_id > { prev_user_id } AND
+            t.secondary1 is not NULL
+            ORDER BY "user_id"
```

Comment:
for explicitness lets specify ASC or DESC, doesnt matter which

Reply:
realized this is already sorted by ASC earlier in the query. confirmed that it works as expected without having any extra sorting

Reply:
oh right, duh